jumpscale/servers/threebot/threebot.py

```diff
@@ -411,6 +432,24 @@ def install(self, package):
         # execute package start method
         package.start()
 
+    def reload(self, package_name):
+        package = self.get(package_name)
+        if not package:
+            raise j.exceptions.NotFound(f"{package_name} package not found")
+
+        # re-install package
+        package.install()
+
+        # install package if threebot is started
+        if self.threebot.started:
```

Comment:
move this check up and if threebot is not started raise an error
reload mean the server is started and you need to reload the package without restarting the server
so reloading the package while the server is not running doesn't make sense